.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,19 @@
 # Contributing
+![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
 
-Please see instructions for contributing here:
-https://checkstyle.org/contributing.html
-
-# Reporting issues
-
-Please read https://checkstyle.org/report_issue.html
+#### [Reporting issues](https://checkstyle.org/report_issue.html)
 
 Please provide issue report in format that we request, EACH DETAIL MAKE A HUGE HELP.
```

Comment:
Please make a link yo page where we explain format

Reply:
Can you please provide the link. I am not aware of it

Reply:
https://github.com/checkstyle/checkstyle/blob/master/.github/PULL_REQUEST_TEMPLATE.md
https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md

https://checkstyle.org/report_issue.html#How_to_report_a_bug

Reply:
Done